### PR TITLE
Update scientific.py

### PR DESCRIPTION
**Pull request by Michael49475**
I added the functions I made and got them working with the buttons. I also fixed the sqrt and the reciprocal buttons, but I use a function for the reciprocal there might be a better way to do it.

### CODE REVIEW

**Review by NGKIPE01 — APPROVED** [review]

File: scientific.py [hunk 1]
```diff
@@ -1 +1,219 @@
...
+
+# start row 3
+
+but_divX = tk.Button(calc, text="1/x", command=lambda: add_to_calculation("reciprocal("), width=5, font=("Arial", 14))
+but_divX.grid(row=3, column=0)
+
+but_xSqr = tk.Button(calc, text="x^2", command=lambda: add_to_calculation(""), width=5, font=("Arial", 14))
+but_xSqr.grid(row=3, column=1)
+
+btnsqrt = tk.Button(calc, text="√", command=lambda: add_to_calculation("sqrt("), width=5, font=("Arial", 14))
```
Comment (Coderdreamgirl) [inline scientific.py:118]: make sure to follow the same pattern as the other buttons. It seems more consistent as a whole.

File: scientific.py [hunk 2]
```diff
@@ -1 +1,219 @@
...
+but_percent = tk.Button(calc, text="%", command=lambda: add_to_calculation("%"), width=5, font=("Arial", 14))
+but_percent.grid(row=2, column=0)
+
+but_clear = tk.Button(calc, text="CE", command=clear_field, width=5, font=("Arial", 14), bg="#7d4e57", fg="#f1faee")
+but_clear.grid(row=2, column=1)
+
+but_delete = tk.Button(calc, text="<=", command=lambda: add_to_calculation(""), width=11, font=("Arial", 14))
+but_delete.grid(row=2, column=2, columnspan=2)
+
+btnfac = tk.Button(calc, text='!', command=lambda: add_to_calculation('factorial('), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:106]:
```suggestion
but_fac = tk.Button(calc, text='!', command=lambda: add_to_calculation('factorial('), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 3]
```diff
@@ -1 +1,219 @@
...
+btnfac.grid(row=2, column=4)
```
Comment (Coderdreamgirl) [inline scientific.py:107]:
```suggestion
but_fac.grid(row=2, column=4)
```

File: scientific.py [hunk 4]
```diff
@@ -1 +1,219 @@
...
+# start row 3
+
+but_divX = tk.Button(calc, text="1/x", command=lambda: add_to_calculation("reciprocal("), width=5, font=("Arial", 14))
+but_divX.grid(row=3, column=0)
+
+but_xSqr = tk.Button(calc, text="x^2", command=lambda: add_to_calculation(""), width=5, font=("Arial", 14))
+but_xSqr.grid(row=3, column=1)
+
+but_sqrt = tk.Button(calc, text="√", command=lambda: add_to_calculation("sqrt("), width=5, font=("Arial", 14))
+btnsqrt.grid(row=3, column=2)
```
Comment (Coderdreamgirl) [inline scientific.py:119]:
```suggestion
but_sqrt.grid(row=3, column=2)
```

File: scientific.py [hunk 5]
```diff
@@ -1 +1,219 @@
...
+
+# button /
+but_div = tk.Button(calc, text="/", command=lambda: add_to_calculation("/"), width=5, font=("Arial", 14))
+but_div.grid(row=3, column=3)
+
+btnln = tk.Button(calc, text='ln', command=lambda: add_to_calculation('ln('), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:125]:
```suggestion
but_ln = tk.Button(calc, text='ln', command=lambda: add_to_calculation('ln('), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 6]
```diff
@@ -1 +1,219 @@
...
+btnln.grid(row=3, column=4)
```
Comment (Coderdreamgirl) [inline scientific.py:126]:
```suggestion
but_ln.grid(row=3, column=4)
```

File: scientific.py [hunk 7]
```diff
@@ -1 +1,219 @@
...
+
+# number 9
+but9 = tk.Button(calc, text="9", command=lambda: add_to_calculation(9), width=5, font=("Arial", 14))
+but9.grid(row=4, column=2)
+
+# button *
+but_multi = tk.Button(calc, text="*", command=lambda: add_to_calculation("*"), width=5, font=("Arial", 14))
+but_multi.grid(row=4, column=3)
+
+btnlog = tk.Button(calc, text='log', command=lambda: add_to_calculation('log('), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:147]:
```suggestion
but_log = tk.Button(calc, text='log', command=lambda: add_to_calculation('log('), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 8]
```diff
@@ -1 +1,219 @@
...
+btnlog.grid(row=4, column=4)
```
Comment (Coderdreamgirl) [inline scientific.py:148]:
```suggestion
but_log.grid(row=4, column=4)
```

File: scientific.py [hunk 9]
```diff
@@ -1 +1,219 @@
...
+# number 3
+but3 = tk.Button(calc, text="3", command=lambda: add_to_calculation(3), width=5, font=("Arial", 14))
+but3.grid(row=6, column=2)
+
+# button +
+but_plus = tk.Button(calc, text="+", command=lambda: add_to_calculation("+"), width=5, font=("Arial", 14))
+but_plus.grid(row=6, column=3)
+
+
+btnpi = tk.Button(calc, text='π', command=lambda: add_to_calculation(math.pi), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:190]:
```suggestion
but_pi = tk.Button(calc, text='π', command=lambda: add_to_calculation(math.pi), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 10]
```diff
@@ -1 +1,219 @@
...
+btnpi.grid(row=6, column=4)
```
Comment (Coderdreamgirl) [inline scientific.py:191]:
```suggestion
but_pi.grid(row=6, column=4)
```

File: scientific.py [hunk 11]
```diff
@@ -1 +1,219 @@
...
+
+btnprime = tk.Button(calc, text='prime', command=lambda: add_to_calculation('testprime('), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:193]:
```suggestion
but_prime = tk.Button(calc, text='prime', command=lambda: add_to_calculation('testprime('), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 12]
```diff
@@ -1 +1,219 @@
...
+btnprime.grid(row=6, column=5)
```
Comment (Coderdreamgirl) [inline scientific.py:194]:
```suggestion
but_prime.grid(row=6, column=5)
```

File: scientific.py [hunk 13]
```diff
@@ -1 +1,219 @@
...
+but0 = tk.Button(calc, text="0", command=lambda: add_to_calculation(0), width=5, font=("Arial", 14))
+but0.grid(row=7, column=1)
+
+but_close = tk.Button(calc, text=")", command=lambda: add_to_calculation(")"), width=5, font=("Arial", 14))
+but_close.grid(row=7, column=2)
+
+but_equals = tk.Button(calc, text="=", command=evaluate_calculation, width=5, font=("Arial", 14))
+but_equals.grid(row=7, column=3)
+
+btne = tk.Button(calc, text='e', command=lambda: add_to_calculation(math.e), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:212]:
```suggestion
but_e = tk.Button(calc, text='e', command=lambda: add_to_calculation(math.e), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 14]
```diff
@@ -1 +1,219 @@
...
+btne.grid(row=7, column=4)
```
Comment (Coderdreamgirl) [inline scientific.py:213]:
```suggestion
but_e.grid(row=7, column=4)
```

File: scientific.py [hunk 15]
```diff
@@ -1 +1,219 @@
...
+
+btncomma = tk.Button(calc, text=',', command=lambda: add_to_calculation(','), width=5, font=('Arial', 14))
```
Comment (Coderdreamgirl) [inline scientific.py:215]:
```suggestion
but_comma = tk.Button(calc, text=',', command=lambda: add_to_calculation(','), width=5, font=('Arial', 14))
```

File: scientific.py [hunk 16]
```diff
@@ -1 +1,219 @@
...
+btncomma.grid(row=7, column=5)
```
Comment (Coderdreamgirl) [inline scientific.py:216]:
```suggestion
but_comma.grid(row=7, column=5)
```

**Review by Coderdreamgirl — CHANGES_REQUESTED** [review]

**Review by Coderdreamgirl — APPROVED** [review]

**Pull request merged**